tests/unit/conftest.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""pytest fixtures for the unit test."""
+
+import contextlib
+import typing
+import unittest.mock
+
+import ops
+import pytest
+from ops.model import BlockedStatus, Container
+from ops.testing import Harness
+
+from tests.unit import helpers
+from tests.unit._patched_charm import DiscourseCharm, pgsql_patch
+
+BLOCKED_STATUS = BlockedStatus.name  # type: ignore
+
+DATABASE_NAME = "discourse"
+
+
+@pytest.fixture(name="start_harness", autouse=True)
+def start_harness_fixture():  # -> typing.Generator[Harness, None, None]:
+    """Ops testing framework harness fixture."""
+
+    def start_harness(
```

Comment:
What is the reason for making this a fixture rather than a helper function please?

Reply:
Changed to a helper function